.github/workflows/apisec-scan.yml

```
@@ -74,6 +74,8 @@ permissions:
 jobs:
 
   Trigger_APIsec_scan:
+    Trigger_APIsec_scan:
```

actionlint (1.7.10)

unexpected key "Trigger_APIsec_scan" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)